### PR TITLE
Add silicon errata from pixhawk.org

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/flight_controller/silicon_errata.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+This page lists known issues with silicon (hardware) errata of 3rd-party parts (micro controller, sensors, etc.) used on the Pixhawk board series. Depending on the type of silicon error, these cannot be fixed in software and might impose specific limitations.
+
+## FMUv2 / Pixhawk Silicon Errata
+
+### STM32F427VIT6 (errata)
+
+Flash Bank 2 and full speed USB device exclusive
+
+Silicon revisions up to rev 1 (revision 3 is the first not affected) can produce errors / data corruption when accessing the 2nd flash bank while there is activity on PA12, which is one of the USB data lines. There is no workaround / software fix for this, except to not use the flash bank #2. Since USB is needed to program the device, Pixhawk revisions built with silicon revisions < rev 3 can only use up to 1MB of the 2MB flash of the microprocessor.
```
Comment (ndepal) [inline en/flight_controller/silicon_errata.md]: > Silicon revisions up to rev 1 (revision 3 is the first not affected)

I don't understand what this means for rev 2?

Comment (hamishwillee) [reply en/flight_controller/silicon_errata.md]: @ndepal That is a very good question - I just copied this from pixhawk.org in order to fix the broken links from some of our tools. @bkueng - can you comment - is rev2 also affected?

Comment (bkueng) [reply en/flight_controller/silicon_errata.md]: Indeed a good question. I really don't know, but `silicon revisions < rev 3` indicates that rev 2 is affected as well.

Comment (hamishwillee) [reply en/flight_controller/silicon_errata.md]: Thanks. I've changed it to cover 1 and 2 in https://github.com/PX4/px4_user_guide/commit/e0247d950815b35e80dbeba523219e9dd51f400d

**Pull request merged**